src/promptflow-devkit/promptflow/_sdk/entities/_chat_group/_chat_role.py

```diff
@@ -108,6 +121,34 @@ def flow(self):
         """Flow definition of the chat role"""
         return self._flow_definition
 
+    @property
+    def environment_variables(self):
+        return self._environment_variables
+
+    @property
+    def log_path(self):
+        return self._log_path
+
+    @property
+    def output_dir(self):
+        return self._output_dir
+
+    @property
+    def worker_count(self):
+        return self._worker_count
+
+    @property
+    def line_timeout_sec(self):
+        return self._line_timeout_sec
+
+    @property
+    def executor_client(self):
```

Comment:
may hide some property if we don't want customer to use

Reply:
updated.